test/suite/test_bug030.py

```diff
@@ -122,4 +122,4 @@ def test_bug030(self):
             evict_cursor.search()
         evict_cursor.reset()
         evict_cursor.close()
-        self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(70))
+        self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(70))
```

Comment:
This change is unnecessary, revert it.

Reply:
Yes, missing last line.

Reply:
Fixed in [d18f392](https://github.com/wiredtiger/wiredtiger/pull/9363/commits/d18f3929bb049cf5a9287d5ca78224379f0aaa3d)